gbif-occurrence-linkage/main.py

```diff
@@ -121,14 +89,20 @@ def run_api_call(specimen_data: Dict) -> Dict[str, str]:
     response = requests.get(query_string)
     response_json = json.loads(response.content)
     if response_json['count'] == 1:
```

Comment:
Make sure the types are also the same, so I would check with '==='. possibly wrapping the count variable in a toNumber function

Reply:
had to look that up, `===` is a javascript thing :laughing: 
That's not necessary because this is a value from the API, it will always be a number unless the API changes, in which case the MAS would need to be updated anyway

Reply:
Ah that is something I forgot over time :D
If it is always a number it should be fine!